clang/lib/Basic/Targets/DPU.cpp

```diff
@@ -32,6 +37,8 @@ void DPUTargetInfo::getTargetDefines(const LangOptions &Opts,
                                      MacroBuilder &Builder) const {
   DefineStd(Builder, "DPU", Opts);
   Builder.defineMacro("__ELF__");
+  Builder.defineMacro("NR_THREADS", STR(DPU_NR_THREADS));
```

Comment:
in the Twine class used by defineMacro(), I'm seeing a constructor from an unsigned long,
doesn't ```Builder.defineMacro("NR_THREADS", DPU_NR_THREADS);``` simply work ?

Reply:
``Builder.defineMacro("NR_THREADS", DPU_NR_THREADS)`` does not work, but:
``Builder.defineMacro("NR_THREADS", Twine(DPU_NR_THREADS))`` do